packages/peregrine/lib/talons/AuthModal/useAuthModal.js

```diff
@@ -36,14 +36,21 @@ export const useAuthModal = props => {
         showMainMenu,
         showMyAccount,
         signOutMutation,
-        view
+        view,
+        handleFormErrors,
+        resetFormErrors
     } = props;
 
     const apolloClient = useApolloClient();
+
     const [isSigningOut, setIsSigningOut] = useState(false);
     const [username, setUsername] = useState('');
     const [{ currentUser, isSignedIn }, { signOut }] = useUserContext();
-    const [revokeToken] = useMutation(signOutMutation);
+    const [revokeToken] = useMutation(signOutMutation, {
+        onError: signOutMutationErrors => {
+            handleFormErrors([signOutMutationErrors]);
```

Comment:
This is not a form error, the error should be destructured from the mutation response object and handled in a toast.